private/src/styles/blocks/core-blocks/_rss.scss

```diff
@@ -12,18 +12,14 @@
 .wp-block-rss__item-title {
   margin-bottom: 12px;
   font-family: var(--wp--preset--font-family--secondary);
-  font-size: map-get($mobile-header-sizes, h4);
+  font-size: var(--wp--preset--font-size--heading-4);
   font-weight: bold;
   line-height: 1.12;
   color: $color-black;
 
   a {
     text-decoration: none !important;
   }
-
-  @include mq(small) {
```

Comment:
We shouldn't need this media query anymore as it's just setting a font-size, and this heading size is now fluid.